src/coprocessor/statistics/analyze.rs

```diff
@@ -297,6 +329,26 @@ impl<S: Snapshot> SampleBuilder<S> {
                 columns_info = &columns_info[1..];
             }
 
+            if self.analyze_common_handle {
+                for logical_row in &result.logical_rows {
+                    let mut data = vec![];
+                    for handle_id in &self.common_handle_col_ids {
+                        let mut handle_col_val = vec![];
+                        columns_slice[*handle_id as usize].encode(
+                            *logical_row,
+                            &columns_info[*handle_id as usize],
+                            &mut EvalContext::default(),
+                            &mut handle_col_val,
+                        )?;
+                        data.extend_from_slice(&handle_col_val);
+                        if let Some(common_handle_cms) = common_handle_cms.as_mut() {
+                            common_handle_cms.insert(&data);
```

Comment:
It seems that we would append `handle_col_val` from previous iteration into `common_handle_cms`. Is this intended?

Reply:
Yes, the same as MockTiKV in TiDB.